controllers/quiz.controller.js

```diff
@@ -0,0 +1,32 @@
+const { Quiz } = require("../models/quiz.model");
+
+const addNewQuiz = async (req, res) => {
+   try {
+      const { loadQuiz } = req.body;
+
+      const quizBank = new Quiz(loadQuiz);
+
+      await quizBank.save();
+
+      res.status(200).json({ message: "found quiz", quiz: quizBank });
+   } catch (error) {
+      res.status(500).json({ message: "could not retrive quiz" });
+   }
+};
+
+const checkLevelAndGetQuizBank = async (req, res) => {
+   try {
+      const { selectedLevel } = req.body;
+      const quizBank = await Quiz.findOne({ level: selectedLevel });
+
+      if (!quizBank) {
+         return res.status(400).json({ message: "quiz not found" });
+      }
+      quizBank.level = undefined;
+      res.status(200).json({ message: "found quiz", quiz: quizBank });
+   } catch (error) {
+      res.status(500).json({ message: "could not retrive quiz" });
```

Comment:
can make a simple asyncErrorHandler so that you can use the common error handler instead of writing try catch again and again. :D